lib/pansophy/remote/create_file.rb

```diff
@@ -11,7 +29,8 @@ def initialize(bucket, path, body)
 
       def call(options = {})
         prevent_overwrite! unless options[:overwrite]
-        directory.files.create(key: @pathname.to_s, body: @body.dup)
+        file_attributes = options.delete_if { |k,v| !ALLOWED_ATTRS.include?(k) }
```

Comment:
Double negative and destructive? Is there a positive and peaceful way of doing it, e.g.:
```ruby
file_attributes = options.select { |k,v| ALLOWED_ATTRS.include?(k) }
```